pkgs/by-name/zu/zulip-term/package.nix

```
@@ -29,24 +29,34 @@ with py.pkgs;
 
 buildPythonApplication rec {
   pname = "zulip-term";
-  version = "0.7.0";
+  version = "2025-05-19";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "zulip";
     repo = "zulip-terminal";
-    rev = "refs/tags/${version}";
-    hash = "sha256-ZouUU4p1FSGMxPuzDo5P971R+rDXpBdJn2MqvkJO+Fw=";
+    rev = "8e5c0357c8746df64ac427d5db3d2cb0f002f975";
+    hash = "sha256-DW3GZ1hY/wZ6P/djPUlAvNIFcBV994FLJ3aiPfDVUBM=";
   };
 
   patches = [
     ./pytest-executable-name.patch
   ];
 
   nativeBuildInputs = with py.pkgs; [
+    pythonRelaxDepsHook
```

```suggestion
  build-system = with py.pkgs; [
```
`pythonRelaxDeps` is automatically added